### DISCUSSION TITLE
How do Jinja Prompts Work? Copy pasting the example code is returning the wrong results.

### DISCUSSION THREAD

**Discussion by Wolvezi** [adieyal/sd-dynamic-prompts#767] [Q&A]
I paste this example into the Jinja2 Templates node, which the [Dynamic Prompts documentation](https://github.com/adieyal/sd-dynamic-prompts/blob/main/jinja2.md) says is supposed to produce three prompts, one for each color, and set batch count to 3:
```
{% for colour in ['red', 'blue', 'green'] %}
    {% prompt %}I love {{ colour }} roses{% endprompt %}
{% endfor %}

```
However, it only outputs `I love red roses`. Did I miss something?

![image](https://github.com/adieyal/sd-dynamic-prompts/assets/166175209/cbc13ce3-1bc7-49db-ad4a-8fca095bacbc)

**Comment by akx** [comment]
This is `sd-dynamic-prompts`, the extension for [a1111 sd webui](https://github.com/AUTOMATIC1111/stable-diffusion-webui); [`comfyui-dynamic-prompts`](https://github.com/adieyal/comfyui-dynamicprompts) is a different beast.

**Reply by Wolvezi** [reply]
I did mean to post on the Comfy version, but I have actually tried it on both with the same results.
